src/main/scala/cromwell/engine/backend/jes/Run.scala

```diff
@@ -8,6 +8,7 @@ import cromwell.engine.backend.jes.Run.{Failed, Running, Success, _}
 import cromwell.engine.db.{JesStatus, JesId, JesCallBackendInfo, DataAccess}
 import cromwell.util.google.GoogleScopes
 import org.slf4j.LoggerFactory
+import DataAccess._
```

Comment:
Hm. I'm also a fan of _not_ minimizing the changes, but updating the variable name with something like `globalDataAccess`. Anything to indicate that the variable in question is actually some sort of imported singleton.

This feeling is based on emotional baggage after feeling all sorts of "wtf is going on"/"where did this come from" pain from Slick imports of variables+implicits, via things like [`import driver.api._`](https://github.com/broadinstitute/cromwell/blob/0d87d1e7ca0f64d82a630144b16da1d59dae811b/src/main/scala/cromwell/engine/db/slick/SlickDataAccess.scala#L102).

Happy to defer to others though, as naming is yet another one of my nemeses.